src/test/scala/uk/gov/homeoffice/rtemailer/model/TemplateFunctionsSpec.scala

```diff
@@ -168,5 +169,14 @@ class TemplateFunctionsSpec extends CatsEffectSuite {
   test("encrypted token test") {
     assertEquals(testChain(TString("abcdefg"), "accountToken"), Right(TString("6d38446695e7c87d8895a2eb388b44ea841bed93236034366c8cc7baef18c21c")))
   }
+
+
+  test("parseDate") {
+    def parseDT(in :String) :DateTime = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZ").parseDateTime(in).withZone(DateTimeZone.UTC)
```

Comment:
is not applyStringFunction function to call for the test ?

Reply:
The test calls `testFunc` which is an alias for `applyFunction` so it is testing the code I changed. That parseDT is to make sure the expectedValue is correct (and has the right tz).

https://github.com/UKHomeOffice/rt-emailer/pull/5/files#diff-52060736b04b129e80e41674e1b7a825bdad7856ef4f9bca66c591df6d83f39eL19